_assets/wallet/token-lists.json

```diff
@@ -0,0 +1,23 @@
+[
+  {
+    "id": "uniswap",
+    "sourceUrl": "https://ipfs.io/ipns/tokens.uniswap.org",
+    "schema": "https://uniswap.org/tokenlist.schema.json"
+  },
+  {
+    "id": "aave",
+    "sourceUrl": "https://raw.githubusercontent.com/bgd-labs/aave-address-book/main/tokenlist.json"
+  },
+  {
+    "id": "kleros",
+    "sourceUrl": "https://t2crtokens.eth.link"
+  },
+  {
+    "id": "superchain",
+    "sourceUrl": "https://static.optimism.io/optimism.tokenlist.json"
+  },
+  {
+    "id": "status-remote",
+    "sourceUrl": "https://raw.githubusercontent.com/status-im/status-go/refs/heads/develop/_assets/wallet/status-tokens-list.json"
```

Comment:
This one is expected to be in the develop branch (until we find a better location for it).

Reply:
I cannot decide whether having this in the same repo is a good idea or not. Just for reference, we used to have a remote list of onramp providers in a separate repo in the past: https://github.com/status-im/crypto-on-ramps. The mechanism for fetching it was basically the same thing you're doing here.

Reply:
Separate repo is a good option for sure.